src/tailwindcss.worker.ts

```diff
@@ -76,17 +76,24 @@ async function stateFromConfig(configPromise: Config | PromiseLike<Config>): Pro
     jitContext,
     separator: config.separator,
     screens: config.theme?.screens ? Object.keys(config.theme.screens) : [],
-    variants: getVariants(jitContext),
+    variants: jitContext.getVariants(),
```

Comment:
nooooooin it took me/us so long to type `getVariants` in ts ^^

but happy to have that code removed

Reply:
I know! I’m not sure if this function already existed before, but we just missed it :see_no_evil: